docs/numeric-types/index.md

```diff
@@ -0,0 +1,264 @@
+# Numeric Types
+
+OCaml's numeric types work differently than in JavaScript. The first thing you
+might notice is that OCaml makes a clear and hard distinction between integer
+and float types. For example:
+
+```reason
+let foo = 42;   // int
+let bar = 42.1; // float
+Js.log(foo);
+Js.log(bar);
+```
+
+Note that you didn't have to write the types yourself, OCaml is able to infer
+the types from the literals. This would also work:
+
+```reason
+let foo: int = 42;
+let bar: float = 42.1;
+Js.log(foo);
+Js.log(bar);
+```
+
+## Melange Playground
+
+[Melange Playground](https://melange.re/v2.0.0/playground/) is an interactive
```

Comment:
This is a nit, but we can use links to v2.1.0 now (applies to other links in the PR)

Reply:
Updated links in https://github.com/melange-re/melange-for-react-devs/pull/2/commits/99e33ffbe4ae012c9300e563738d62239cc0bb7e